src/recursion.js

```diff
@@ -1,16 +1,30 @@
 // Complete the following functions.
-
 const nFibonacci = (n) => {
+	if (n <= 1) {
+		return 1
+	}
+	return n * nFibonacci(n-1);
   // fibonacci sequence: 1 2 3 5 8 13 ...
   // return the nth number in the sequence
-};
+}; 
 
 const nFactorial = (n) => {
+	if (n <= 1) {
+		return true;
+	}
+	return nFactorial(n - 1) + nFactorial(n - 2);
   // factorial example: !5 = 5 * 4 * 3 * 2 * 1
   // return the factorial of `n`
 };
 
 const checkMatchingLeaves = (obj) => {
+	  if (obj.property) {
```

Comment:
I know you don't have a finished solution here but it's always good to mind the indentation, even with incomplete code.